tests/ComTests.cpp

```diff
@@ -1452,7 +1452,7 @@ void TestSmartPointerQueryIidPpv(wistd::true_type, const Ptr& source)       // i
         if (source)
         {
             DestPtr dest;
-            source.query_to(IID_PPV_ARGS(&dest));
+            REQUIRE_NOERROR(source.query_to(IID_PPV_ARGS(&dest)));

```

Comment:
Note that these changes were made before I reverted the updates to `query_to` et. al. I kept these checks because they seemed good regardless